dds/DCPS/ValueWriter.h

```diff
@@ -189,7 +195,7 @@ class OpenDDS_Dcps_Export ValueWriter {
   ///@}
 
 protected:
-  std::string bit_string(ACE_CDR::ULongLong value, ACE_CDR::ULong bitbound);
+  String bitflag_string(ACE_CDR::ULongLong value, const BitmaskHelper& helper);
```

Comment:
Could the name of the function be `bitmask_string` or `bitmask_to_string`?

Reply:
`bitmask_to_string` sounds good to me.